bin/main

```diff
@@ -1,131 +1,71 @@
 #!/usr/bin/env ruby
+<<<<<<< HEAD
+=======
+require '../lib/board.rb'
+require '../lib/player.rb'
+require '../lib/winning.rb'
+require '../lib/playagain.rb'
```

Comment:
These should be `require_relative`. `require` should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory.